airgun/entities/eol_banner.py

```diff
@@ -0,0 +1,32 @@
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.eol_banner import EOLBannerView
+
+
+class EOLBannerEntity(BaseEntity):
+    def read(self):
+        view = self.navigate_to(self, 'NavigateToEOLBanner')
+        return view.read()
+
+    def dismiss(self):
+        view = self.navigate_to(self, 'NavigateToEOLBanner')
+        view.dismiss_button.click()
+
+    def is_warning(self):
+        view = self.navigate_to(self, 'NavigateToEOLBanner')
+        return view.warning
+
+    def is_danger(self):
+        view = self.navigate_to(self, 'NavigateToEOLBanner')
+        return view.danger
+
+
+@navigator.register(EOLBannerEntity)
+class NavigateToEOLBanner(NavigateStep):
+    VIEW = EOLBannerView
+
+    def step(self, *args, **kwargs):
```

Comment:
Is this information intended for display on the dashboard? I couldn't locate any actionable steps or a URL path to navigate to the corresponding entity. If this is the case, how do we define this entity? Shouldn't this be handled within the dashboard page entity itself?

Reply:
Hm, it doesn't feel right to me, because it would seem that the banner is something tied to the dashboard, but it really is a thing of its own. It's just not big enough to have a component of its own, therefore it is tracked under dashboard component. Hope this makes sense